bpfd/src/bpf.rs

```diff
@@ -354,6 +439,15 @@ impl BpfManager {
                 let kprobe: &mut KProbe = raw_program.try_into()?;
                 kprobe.load()?;
 
+                p.set_kernel_info(kprobe.program_info()?.try_into()?);
```

Comment:
Why do we need to call `set_kernel_info()` twice for kprobe and uprobe? For kprobe, it's called here and line 460. Similar pattern in uprobe.